lib/pf/radius/rest.pm

```diff
@@ -71,7 +71,7 @@ sub format_request {
     my ($request) = @_;
     # transform the request according to what radius_authorize expects
     my %remapped_radius_request = map {
-        $_ => ( exists $ALLOWED_MULTIPLE{$_} ) ? $request->{$_}{value} : $request->{$_}{value}[0]
```

Comment:
Should we just remove the ALLOWED_MULTIPLE definition then or is it used elsewhere?